src/frontend/src/components/Header/index.tsx

```diff
@@ -6,30 +6,22 @@ import { Link } from "react-router-dom";
 export default class Header extends React.Component {
   public render() {
     return (
-      <nav className="center pt3 ph2 bg-white shadow">
-        <div className="mw8 center flex-l justify-between">
-          <Link to="/">
-            <div className="logo mb3 mb0-l">
-              <Logo />
-            </div>
-          </Link>
-
-          <div className="dib mb3">
-            <button
-              onClick={() => history.push("/record-search")}
-              className="link mid-gray hover-blue f6 f5-ns dib pa3"
-            >
-              Search
-            </button>
-            <button
-              onClick={() => history.push("/manual")}
-              className="link mid-gray hover-blue f6 f5-ns dib pa3"
-            >
+      <div className="bg-white shadow">
+        <nav className="mw8 relative center flex flex-wrap justify-between pa3">
+          <div className="logo mb4 mb0-ns">
+            <a href="/"><Logo /></a>
+          </div>
+          <div className="mt5 mt2-ns">
+            <a href="/manual" className="link hover-blue f5 fw6 pv2 ph0 ph3-ns mr4-ns">
               Manual
-            </button>
+            </a>
+            <a href="/record-search" className="absolute top-1 right-1 static-ns bg-blue white bg-animate hover-bg-dark-blue f5 fw6 br2 pv2 ph3">
```

Comment:
@wittejm We need to follow up and change this to use `<Link to="...">` (which internally uses history.push) as this is an internal link. See https://stackoverflow.com/a/55456045/2750819.